health/notifications/alarm-notify.sh.in

```diff
@@ -1887,8 +1887,8 @@ send_syslog() {
 
     message="${prefix} ${status} on ${host} at ${date}: ${chart} ${value_string}"
 
-    if [ ${host} ]; then
-      logger_options="${logger_options} -n ${host}"
+    if [ ${server} ] && [ -z $temp1 ]; then
```

Comment:
Why do we need `&& [ -z $temp1 ]`? I this if there is `${server}` then we need to send the message to the remote host, isn't `${server}` enough?

To be honest i don't really follow all this magic with temp1/temp2/etc 😅

Reply:
the `tempX` variable is a little bit confused, we will need another PR to rename them. 

The comparison I added it is not more necessary after to remove the conditional with the previous commit. I removed it.